src/endpoint/s3/ops/s3_get_service.js

```diff
@@ -15,7 +15,7 @@ async function list_buckets(req) {
             Buckets: reply.buckets.map(bucket => ({
                 Bucket: {
                     Name: bucket.name.unwrap(),
-                    CreationDate: date,
+                    CreationDate: s3_utils.format_s3_xml_date(bucket.creation_date) || date,
```

Comment:
does format_s3_xml_date returns undefined if the passed date is undefined? otherwise this fallback || will not workout...